openproblems/tasks/cell_cell_communication/api.py

```diff
@@ -17,6 +18,12 @@
 CCC_PRED_COLUMNS = CCC_COLUMNS.union({"score"})
 
 
+# Helper function to split complex subunits
+def decomplexify(entities):
+    entity_list = [np.chararray.split(x, sep="_").tolist() for x in entities]
```

Comment:
This is confusingly written. Is the intention here to do something like
```
entity_list = [entity for entity_complex in entities for entity in entity_complex.split("_")]
```
?

Reply:
Indeed, looks much simpler. Will change it.

Reply:
OK. Tried this your suggestion, however, when there is a single element in the list, Python simply assumes that it should iterate over the strings and concats them all into a single string.

Reply:
I don't quite understand the failure mode.

![image](https://user-images.githubusercontent.com/84813314/183690793-f1cd0863-05ea-4168-9380-e1842620e02a.png)

Reply:
Maybe you need a `if isinstance(entities, str): entities = [entities]`?

Reply:
Sorry, it does work - have now pushed.